Add distance walked, calories burned, water drink and  sleeping hours.

ramonsan opened the pull request.

ramonsan committed:
Add distanceWalkingRunning, activeEnergyBurned and dietaryWater

ramonsan committed:
Add sleepAnalysis

ramonsan committed:
Add Distance, Calories expended and Hydratation

ramonsan committed:
Add sleep analysis on Android

ramonsan committed:
Updated README.md

krokyze committed:
Update README.md

krokyze committed:
upgrade packages

krokyze committed:
rename

krokyze committed:
remove sleep for this merge

krokyze committed:
activeEnergyBurned in kilocalories

krokyze committed:
remove sleeping hours from readme

krokyze reviewed (approved):
@ramonsan Hey. I made some changes and sorry I removed sleep for this merge. Please make another pull request with sleep functionality :)

ramonsan commented:
> @ramonsan Hey. I made some changes and sorry I removed sleep for this merge. Please make another pull request with sleep functionality :)

Added in Pull Request #9
